hamilton/src/demo_node.cpp

```diff
@@ -0,0 +1,47 @@
+// //TOCHECK 
+// #ifdef __i386__
+//   #pragma message("i386 Architecture detected, disabling EIGEN VECTORIZATION")
+//   #define EIGEN_DONT_VECTORIZE
+//   #define EIGEN_DISABLE_UNALIGNED_ARRAY_ASSERT
+// #else
+//   #pragma message("64bit Architecture detected, enabling EIGEN VECTORIZATION")
+// #endif
+
+#include "hamilton/hybrid_planner.h"
+
+int main(int argc,char** argv)
+{
+  ros::init(argc,argv,"HybridPlanner");
+  ros::AsyncSpinner spinner(2);
+  spinner.start();
+
+  // creating hybrid_planner_
+  hamilton::HybridPlanner hybrid_planner_;
+
+  // loading parameters
+  hybrid_planner_.loadParameters();
+
+  // initializing ros components and Descartes
+  hybrid_planner_.init();
```

Comment:
I think having separate calls to load parameters and initialization is unnecessary, you could do both these tasks inside an initialize method.